src/collections/_documentation/workflow/notifications/alerts.md

```diff
@@ -3,22 +3,129 @@ title: 'Alerts'
+
+[{% asset notifications/alert_environment.png alt="Dropdown of environments. Currently, only All Environments is an option." %}]({% asset notifications/alert_environment.png @path %})
+
+## Digests
+
+Sentry will automatically digest alerts sent by some services to avoid flooding your inbox with individual issue notifications. To control how frequently notifications are delivered, use the sliders.
+
+- Minimum delivery interval
+- Maximum delivery interval
+
+[{% asset notifications/alert_digest.png alt="Sliding time scale for intervals of notifications." %}]({% asset notifications/alert_digest.png @path %})
+
+## Notification Management
+
+You can also configure various personal notifications from your [personal notifications settings](https://sentry.io/settings/account/notifications) and any specific project's settings in the Project Settings page.
+
+### Default Project Alerts
+
+If you have set any specific project alert to default, that means you've configured the notification to the Default Project Alert, which is a switch you can toggle on or off. This toggle switch controls all project alerts set to default. 
+
+You can find the Default Project Alerts switch in **User Settings > Account > Notifications > Fine tune alerts by project**.
+
+[{% asset notifications/default_project_alerts.png alt="Toggle for turning defaul project alerts on and off." %}]({% asset notifications/default_project_alerts.png @path %})
+
+### Per-Project Alerts
+
+Your per-project alerts have the fine-tuning options of Default, On, or Off. 
+
+[{% asset notifications/specific_project_alert.png alt="Dropdown for per-project alerts. Options include default, on, or off." %}]({% asset notifications/specific_project_alert.png @path %})
+
+## Issue Owners
+
+The Issue Owners page is located in **Project Settings > Issue Owners**.
+
+If an issue owner is specified, the email notification will go to the issue owner. If no issue owner is specified, all project members are notified.
+
+[{% asset notifications/issue_owners.png alt="Toggles for turning issue owner notifications on and off." %}]({% asset notifications/issue_owners.png @path %})
+
+For more details, see full documentation on [Issue Owners]({%- link _documentation/workflow/issue-owners.md -%}).
+
+## FAQs
+
+- Can I copy an alert rule to another project?
+    - Unfortunately, this is not an option.
+- Can I set different default alert rules?
+    - Unfortunately, this is not an option.
+- Are there issue-level filters?
+    - No, all filters are event-based. For example, there aren't configurations for alerting only if an issue is X days old, or assigned to Y, or alerted-on-before, etc.
+- What is the difference between Delete, Delete & Discard, and Ignore
+    - Delete - when you delete an issue with notifications, notifications will continue if the issue is triggered again
+    - Delete & Discard - when you delete and discard, all notifications will stop
```

Comment:
could we link to this (admittedly sparse) [delete and discard](https://docs.sentry.io/accounts/quotas/#filter-by-issue) documentation?